### ISSUE TITLE
fallback on ios8

### ISSUE THREAD

**Issue by funkyLover** [roytang121/Akaibu#1]
I try Akaibu in my project, get an error at 

``` swift
if let data = try NSKeyedUnarchiver.unarchiveTopLevelObjectWithData(obj) as? Akaibu {
   return data
}
```

and xcode replace the snippet like that

``` swift
if #available(iOS 9.0, *) {
     if let data = try NSKeyedUnarchiver.unarchiveTopLevelObjectWithData(obj) as? Akaibu {
          return data
     }
} else {
   // Fallback on earlier versions
}
```

I need to support ios8. is there a way fallback on ios8?

**Comment by roytang121** [comment]
fixed with #b18b721f1a16091789f13e5cbebb7d34af81e6da.
Thank you for the issue report : )

**Comment by funkyLover** [comment]
cool!
